cmd/coder/sync.go

```diff
@@ -79,9 +75,7 @@ func (cmd *syncCmd) Run(fl *pflag.FlagSet) {
 		err = s.Run()
 	}
 
-	if err == NoRsync {
-		flog.Fatal("no compatible rsync present on remote machine")
-	} else {
+	if err != nil {
```

Comment:
As seconded by @scsmithr we don't want to be checking errors this way; logic moved to internal/sync.go and now checks the process exit codes instead.